package/AUTHORS

```diff
@@ -209,6 +209,7 @@ Chronological list of authors
   - Vishal Parmar
   - Moritz Schaeffler
   - Xu Hong Chen
+  - DrDomenicoMarson
```

Comment:
We would prefer your real name rather than github handle here, but no obligation :)

Reply:
I edited it, I also prefer the real name, I just didn't notice real names were used :-)